lib/state/appState.js

```diff
@@ -59,6 +59,11 @@ appState.getInitial = function() {
       name: 'OneTouch UltraMini',
       key: 'onetouchmini',
       source: {type: 'device', driverId: 'OneTouchMini'}
+    },
+    {
+      name: 'BayerContourNext',
```

Comment:
Can you put spaces in here - `Bayer Contour Next`? Although this makes me realize we should refactor so the display name is only in one place. (This one gets displayed in the device list, the one from `deviceInfo.js` gets displayed in the upload list.) I'll put the refactor on my TODO. ;)